index.html

```diff
@@ -250,99 +243,148 @@ <h2 class="tonic">
               Get my resume
             </button>
           </div>
           <div class="technologies">
-              <ul class="educ langguages">
-                 <li>Languages</li>
-                  <li><ion-icon name="chevron-down-outline"></ion-icon></li>
+            <ul class="educ langguages">
+              <li>Languages</li>
+              <li>
+                <ion-icon name="chevron-down-outline">
+                </ion-icon>
+              </li>
+            </ul>
+            <div class="3langguages">
+              <ul class="langguage">
+                <li><img class="ellipse"
+                    src="images/Ellipse-1.png"
+                    alt="Ellipse 1"></li>
+                <li class="name">JavaScript</li>
               </ul>
-              <div class="3langguages">
-                   <ul class="langguage">
-                      <li><img class="ellipse" src="images/Ellipse-1.png" alt="Ellipse 1"></li>
-                       <li class="name">JavaScript</li>
-                   </ul>
-                   <ul class="langguage">
-                      <li><img class="ellipse" src="images/Ellipse-2.png" alt="Ellipse 2"></li>
-                       <li class="name">HTM</li>
-                   </ul>
-                   <ul class="langguage">
-                      <li><img class="ellipse" src="images/Ellipse-3.png" alt="Ellipse 3"></li>
-                       <li class="name">CSS</li>
-                   </ul>
-              </div>
-              <ul class="educ separator frameworks">
-                  <li>Frameworks</li>
-                 <li><ion-icon name="chevron-forward-outline"></ion-icon></li> 
+              <ul class="langguage">
+                <li><img class="ellipse"
+                    src="images/Ellipse-2.png"
+                    alt="Ellipse 2"></li>
+                <li class="name">HTM</li>
               </ul>
-              <hr>
-              <ul class="educ separator skills">
-                  <li>Skills</li>
-                 <li><ion-icon name="chevron-forward-outline"></ion-icon></li>
+              <ul class="langguage">
+                <li><img class="ellipse"
+                    src="images/Ellipse-3.png"
+                    alt="Ellipse 3"></li>
+                <li class="name">CSS</li>
               </ul>
-               <hr>
+            </div>
+            <ul class="educ separator frameworks">
+              <li>Frameworks</li>
+              <li>
+                <ion-icon name="chevron-forward-outline">
+                </ion-icon>
+              </li>
+            </ul>
+            <hr>
+            <ul class="educ separator skills">
+              <li>Skills</li>
+              <li>
+                <ion-icon name="chevron-forward-outline">
+                </ion-icon>
+              </li>
+            </ul>
+            <hr>
           </div>
         </div>
       </div>
     </section>
-</main>
+
+  </main>
+  <footer>
+    <div class="footer-text">
```

Comment:
- [ ] The alignment of the background picture in the original website is a little down, Please fix it.
- Original website
![original1_LI](https://user-images.githubusercontent.com/26767383/149020714-a1eb66cd-af90-4087-82de-09fad258f1d1.jpg)

- Your Website
![original2](https://user-images.githubusercontent.com/26767383/149020033-dd275d89-e184-4d76-af6a-1920740596e7.PNG)

Reply:
Hi @Deepakdanger,
Thank you for your review.